uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-widget/leave-widget-absence-types-available-balance.component.js

```diff
@@ -26,6 +25,8 @@ define([
     var entitlements;
     var vm = this;
 
+    vm.leaveEntitlements = [];
```

Comment:
this name has no meaning really (see below, i'm making the same point somewhere else), a `leaveEntitlement` is not a real thing